helpers.go

```diff
@@ -42,6 +42,34 @@ func getCmd(name string, f func(ctx *cli.Context)) func(*cli.Context) {
 	}
 }
 
+// Returns a string containing the global flags being used.
+func globalFlags() string {
+	if currentContext == nil {
+		return ""
+	}
+
+	res := ""
+	flags := []string{"non-interactive", "no-gpg-checks", "gpg-auto-import-keys"}
+
+	for _, v := range flags {
+		if currentContext.GlobalBool(v) {
+			res = res + "--" + v + " "
+		}
+	}
+	return res
+}
+
+// Concatenate the given zypper commands, while adding the global flags
+// currently in place.
+func formatZypperCommand(cmds ...string) string {
+	flags := globalFlags()
+
+	for k, v := range cmds {
+		cmds[k] = "zypper " + flags + v
+	}
+	return strings.Join(cmds, " && ")
```

Comment:
This example by the way reminds me that I should update that zypper call in this patch too :)